skorch/tests/test_net.py

```diff
@@ -178,6 +178,48 @@ def test_net_init_two_unknown_argument(self, net_cls, module_cls):
                     "should deal with the new arguments explicitely.")
         assert e.value.args[0] == expected
 
+    def test_net_init_missing_dunder_in_prefix_argument(
```

Comment:
We can `pytest.mark.parametrize` and test for some other prefixes like `optimizer_momentum`:

```py
@pytest.mark.parametrize("key, value", 
        [("iterator_train_shuffle", True), ("optimizer_momentum", 0.95)]
```